liquidator/liquidator.js

```diff
@@ -51,17 +52,19 @@ class Liquidator {
         address: position.sponsor
       });
 
-      // Create the liquidation transaction
-
-      // TODO calculate the amountToLiquidate as a function of the total collateral within the position
-      // and the current price of the collateral. This will require knowing how much the collateral and the
-      // synthetic are worth.
+      // Create the liquidation transaction to liquidate the entire position:
+      // - Maximum (Collateral/Synthetic) ratio to liquidate.
+      let liquidateCollateralPerToken = (parseFloat(position.amountCollateral) / parseFloat(position.numTokens));
```

Comment:
Shouldn't we set the collateral per token to the price input instead of dividing these two values?

Reply:
on second thought this is correct. originally my logic was to set the `amountToLiquidate` as a function of the input price when initiating the liquidation. this still needs to happen but the `liquidateCollateralPerToken` should indeed be the price given to the liquidator.

Reply:
hm yeah that makes sense, since you are determining which positions are under collateralized positions based on the `priceFeed`, you also should be liquidating using that `priceFeed`

Reply:
Pushed changes for this. Also added a warning for user to remember the `--price` flag, which is neccessary until we have a more robust price feed module